.azure_pipeline.yml

```diff
@@ -125,10 +125,10 @@ stages:
       vmImage: macOS-10.14
       matrix:
         # MacOS environment with OpenMP installed through homebrew
-        py35_conda_homebrew_libomp:
+        py36_conda_homebrew_libomp:
           PACKAGER: 'conda'
-          VERSION_PYTHON: '3.5'
-          BLAS: 'mkl'
+          VERSION_PYTHON: '3.6'
+          BLAS: 'openblas'
```

Comment:
So the issue with mkl is not only on python 3.5 ?
It's a general issue with OpenMP installed from homebrew ? Because the setup just below seems to work with OpenMP installed from conda-forge.

Reply:
It's related to the Python version. It's probably dependent on the version of libomp and maybe MKL.